### PR TITLE
Adding CFT scanning support for the resources:

### PR DESCRIPTION
**Pull request by shreyas-phansalkar-189**
- aws_qldb_ledger
- aws_ecs_cluster
- aws_apigatewayv2_api
- aws_db_event_subscription
- aws_neptune_cluster_instance
- aws_globalaccelerator_accelerator
- aws_waf_size_constraint_set
- aws_athena_workgroup

### CODE REVIEW

File: pkg/mapper/iac-providers/cft/config/api-gatewayv2-api.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	ApiKeySelectionExpression string                   `json:"api_key_selection_expression,omitempty"`
+	DisableExecuteApiEndpoint bool                     `json:"disable_execute_api_endpoint,omitempty"`
+	FailOnWarnings            bool                     `json:"fail_on_warnings,omitempty"`
+	CorsConfiguration         []CorsConfigurationBlock `json:"cors_configuration,omitempty"`
+}
+
+// GetAPIGatewayV2ApiConfig returns config for aws_apigatewayv2_api resource
+func GetAPIGatewayV2ApiConfig(a *apigatewayv2.Api) []AWSResourceConfig {
+
+	corsConfigData := make([]CorsConfigurationBlock, 1)
```
Comment (gaurav-gogia) [inline pkg/mapper/iac-providers/cft/config/api-gatewayv2-api.go:51]: please add null check here

File: pkg/mapper/iac-providers/cft/config/db-event-subscription.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	Config
+	SnsTopicArn     string   `json:"sns_topic"`
+	Enabled         bool     `json:"enabled,omitempty"`
+	EventCategories []string `json:"event_categories,omitempty"`
+	SourceIds       []string `json:"source_ids,omitempty"`
+	SourceType      string   `json:"source_type,omitempty"`
+}
+
+// GetDBEventSubscriptionConfig returns config for aws_db_event_subscription resource
+func GetDBEventSubscriptionConfig(db *rds.EventSubscription) []AWSResourceConfig {
```
Comment (gaurav-gogia) [inline pkg/mapper/iac-providers/cft/config/db-event-subscription.go:32]: prefer using single letter names like its being done with every other resource

File: pkg/mapper/iac-providers/cft/config/ecs-cluster.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	ClusterSettings                 []ClusterSettingsBlock          `json:"settings"`
+	DefaultCapacityProviderStrategy []CapacityProviderStrategyBlock `json:"default_capacity_provider_strategy"`
+	Configuration                   []ConfigurationBlock            `json:"configuration"`
+}
+
+// GetEcsClusterConfig returns config for aws_ecs_cluster resource
+func GetEcsClusterConfig(e *ecs.Cluster) []AWSResourceConfig {
+
+	clusterSettingsData := make([]ClusterSettingsBlock, len(e.ClusterSettings))
+	for i := 0; i < len(e.ClusterSettings); i++ {
```
Comment (gaurav-gogia) [inline pkg/mapper/iac-providers/cft/config/ecs-cluster.go:68]: we can use `range` here to make code cleaner

**Review by gaurav-gogia — CHANGES_REQUESTED** [review]

File: pkg/mapper/iac-providers/cft/config/ecs-cluster.go (hunk 3 above)
Comment (shreyas-phansalkar-189) [reply pkg/mapper/iac-providers/cft/config/ecs-cluster.go:68]: Wouldnt we need i for indexing the clusterSettingsData array ?

Comment (gaurav-gogia) [reply pkg/mapper/iac-providers/cft/config/ecs-cluster.go]: Try this:
```go
for i := range e.ClusterSettings {
.....
}
```

**Review by gaurav-gogia — APPROVED** [review]

File: pkg/mapper/iac-providers/cft/config/ecs-cluster.go [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+func setConfigurationBlock(e *ecs.Cluster) []ConfigurationBlock {
+	configurationData := make([]ConfigurationBlock, 1)
+	configurationData[0].ExecuteCommandConfig = setExecCommandConfigBlock(e)
+
+	return configurationData
+}
+
+func setExecCommandConfigBlock(e *ecs.Cluster) []ExecuteCommandConfiguration {
+
+	execCommandConfigData := make([]ExecuteCommandConfiguration, 1)
```
Comment (gaurav-gogia) [inline pkg/mapper/iac-providers/cft/config/ecs-cluster.go:108]: Please add null check

File: pkg/mapper/iac-providers/cft/config/ecs-cluster.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+	execCommandConfigData[0].KmsKeyID = e.Configuration.ExecuteCommandConfiguration.KmsKeyId
+	execCommandConfigData[0].Logging = e.Configuration.ExecuteCommandConfiguration.Logging
+	execCommandConfigData[0].LogConfiguration = setLogConfigurationBlock(e)
+
+	return execCommandConfigData
+}
+
+func setLogConfigurationBlock(e *ecs.Cluster) []LogConfigurationBlock {
+
+	logConfigData := make([]LogConfigurationBlock, 1)
```
Comment (gaurav-gogia) [inline pkg/mapper/iac-providers/cft/config/ecs-cluster.go:119]: Please add null check

File: pkg/mapper/iac-providers/cft/config/waf-size-constraint-set.go [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+	return []AWSResourceConfig{{
+		Resource: cf,
+		Metadata: w.AWSCloudFormationMetadata,
+	}}
+}
+
+func setSizeConstraintSet(w *waf.SizeConstraintSet) []SizeConstraintSetBlock {
+	sizeConstraintSet := make([]SizeConstraintSetBlock, len(w.SizeConstraints))
+
+	for i := 0; i < len(w.SizeConstraints); i++ {
```
Comment (gaurav-gogia) [inline pkg/mapper/iac-providers/cft/config/waf-size-constraint-set.go:62]:
```go
for i := range w.SizeConstraints {}
```
this could be cleaner

**Review by gaurav-gogia — CHANGES_REQUESTED** [review]

**Review by gaurav-gogia — APPROVED** [review]

**Pull request merged**